code/src/Apis/Urls.js

```diff
@@ -0,0 +1,2 @@
+export const API_KEY = ('0b0afb2c40259c4645991dad5ac68712')
```

Comment:
I like that you add the API_KEY in a separate file. It makes it easier if you want to hide it and include this in the gitignore file. Since we already have a file for APIs, (my preference) I would save the two api links in this file in 2 variables such as ALL_MOVIES_API and DETAIL_PAGE_API (just examples).